launchdarkly-flags-sync/package.json

```diff
@@ -0,0 +1,26 @@
+{
+  "name": "launch-darkly-feature-flags-sync-action",
+  "version": "1.0.0",
+  "description": "",
+  "main": "src/index.js",
+  "scripts": {
+    "build": "ncc build"
+  },
+  "keywords": [
+    "GitHub",
+    "Actions",
+    "JavaScript"
+  ],
+  "author": "",
+  "license": "MIT",
+  "dependencies": {
+    "@actions/core": "^1.10.0",
+    "@actions/github": "^5.1.1",
```

Comment:
Doesn't look like this one is used so can remove

Reply:
removed